modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/extension/TestRuntimeExtensionModelProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.activation.internal.extension;
+
+import org.mule.runtime.api.meta.model.ExtensionModel;
+import org.mule.runtime.core.api.extension.RuntimeExtensionModelProvider;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class TestRuntimeExtensionModelProvider implements RuntimeExtensionModelProvider {
+
+  @Override
+  public ExtensionModel createExtensionModel() {
```

Comment:
what is this testing?

Reply:
This is needed for `DefaultExtensionModelDiscovererTestCase`, in order to test the discovery of the Runtime extension models.